src/stacker/ConcreteStacker.ts

```diff
@@ -25,6 +25,27 @@ export class ConcreteStacker implements Stacker {
     this.repositoryUpdateListener(repo);
   };
 
+  private localRefPrefix : string = "refs/heads/";
```

Comment:
This should just be a constant outside the class actually. All the util functions below should probably be outside their classes too